### PR TITLE
fixes bannerimage

### PR DESCRIPTION
**Pull request by shilpi1958**

### CODE REVIEW

**Comment by shreyasbapat** [comment]
There are conflicts!

**Comment by shilpi1958** [comment]
i told you there would be

**Comment by shreyasbapat** [comment]
Ok, now comes the chance of this PR

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <li><a href="#">About</a></li>
         <li><a href="#">Team</a></li>
         <li><a href="#">Blog</a></li>
-        <li><a href="#" class="btn black-text" style="background-color: #87f4e2;">Contact Us</a></li>
```
Comment (shreyasbapat) [inline index.html:37]: You never ever ever add styles in HTML :p

**Comment by shilpi1958** [comment]
https://github.com/shilpi1958/furrytails.in/blob/master/images/Resources/Furrytail_website_template%20%5BRecovered%5D-08.png

**Comment by shreyasbapat** [comment]
Now there are more merge conflicts. You need to rebase

**Comment by shreyasbapat** [comment]
> https://github.com/shilpi1958/furrytails.in/blob/master/images/Resources/Furrytail_website_template%20%5BRecovered%5D-08.png

This link pops up a 404

**Comment by shilpi1958** [comment]
yep, I categorized images since you could not find them. New link
https://github.com/shilpi1958/furrytails.in/blob/master/images/Resources/laptop/Furrytail_website_template-04.svg

**Comment by shreyasbapat** [comment]
Oh you are working on your master.

**Comment by shreyasbapat** [comment]
If you work on master, you will always face 10X conflicts.

**Comment by shilpi1958** [comment]
rebasing isn't helping

File: index.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
+        <li><a href="#">Contact Us</a></li>
```
Comment (shreyasbapat) [inline index.html:37]: Don't you think this is not what you designed? Are you sure?

**Comment by shreyasbapat** [comment]
> rebasing isn't helping

Always use new branches for proposing any changes to a project. Never make two pull requests using the same branch. I will try to sort this out for you now with your permission. Just that I don't know which one you want.

File: index.html (hunk 2 above)
Comment (shilpi1958) [reply index.html:37]: yes! looks cleaner

**Comment by shilpi1958** [comment]
sorted!

**Comment by shreyasbapat** [comment]
Untill then, deploy this so that we can see how it looks like

**Comment by shilpi1958** [comment]
its deployed
https://shilpi1958.github.io/furrytails.in/#

**Comment by shreyasbapat** [comment]
I kind of liked that contact us button.

**Comment by shreyasbapat** [comment]
Are you sure you want to merge? This is way different than the design you gave me

**Comment by shreyasbapat** [comment]
Problem, the banner does not span the screen in mobile devices: 
Screenshot from 2020-06-12 13-13-35

**Comment by shilpi1958** [comment]
Merge this Abhi I guess, mobile wala I will handle. the shadow of the button was weird, that's why I removed it. I tried changing it, but couldn't.

Baki your wish

**Comment by shreyasbapat** [comment]
Shadow can be removed. See the documentation of MaterialiseCSS

**Comment by shreyasbapat** [comment]
Now don't make any other Pull Request from master

**Comment by shilpi1958** [comment]
> Shadow can be removed. See the documentation of MaterialiseCSS

I removed for the nav, for the button it wasn't working for me

**Comment by shreyasbapat** [comment]
> > Shadow can be removed. See the documentation of MaterialiseCSS
> 
> I removed for the nav, for the button it wasn't working for me

But your design had a shadow on nav. Why are we moving away from your design?

**Comment by shreyasbapat** [comment]
Screenshot from 2020-06-12 13-24-18

This is your design which is currently in the resources folder

**Comment by shilpi1958** [comment]
I know.

**Comment by shilpi1958** [comment]
image

**Comment by shilpi1958** [comment]
> > > Shadow can be removed. See the documentation of MaterialiseCSS
> > 
> > 
> > I removed for the nav, for the button it wasn't working for me
> 
> But your design had a shadow on nav. Why are we moving away from your design?

It was too dark, I did not remove it , edited it, its lighter now

**Comment by shilpi1958** [comment]
Bhai , theek se dekh le tu pehle sab kuch, phir bata kya load hai :3

**Pull request merged**